### ISSUE TITLE
Browser warning on evt.preventDefault()

### ISSUE THREAD

**Issue by haf** [LouisBrunner/dnd-multi-backend#10]
Getting

```
Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
```

Here:

<img width="538" alt="screen shot 2017-08-21 at 00 01 36" src="https://user-images.githubusercontent.com/193115/29498826-f27e763c-8603-11e7-8337-8d95366fa9ae.png">

With code from https://github.com/react-dnd/react-dnd/blob/master/examples/04%20Sortable/Cancel%20on%20Drop%20Outside/Container.js and this project enabled, like so

```js
// @flow
import { DragDropContext, DragDropContextProvider } from 'react-dnd';
// import HTML5Backend from 'react-dnd-html5-backend';
import MultiBackend from 'react-dnd-multi-backend';
import HTML5toTouch from 'react-dnd-multi-backend/lib/HTML5toTouch'; // or any other pipeline

export const Backend = MultiBackend(HTML5toTouch);
export { DragDropContextProvider as Provider }
export default (App: any) => DragDropContext(Backend)(App);
```

**Comment by LouisBrunner** [comment]
Hi @haf,

Thank you for the report. I had already noticed this warning but didn't think I could do anything about it.

However after a bit of research, the problem comes from the event creation after the backend was switched, so it should happen only once (for one Drag'n'Drop event).

Turnouts this is linked with #11, I'll see if I can avoid doing this kind of dirty hack.

**Comment by LouisBrunner** [comment]
After some investigation, I can't seem to make the warning disappear. The `preventDefault` call happens in [react-dnd-touch-backend](https://github.com/yahoo/react-dnd-touch-backend) in the function that handles `touchmove` events. However if tested alone, `react-dnd-touch-backend` doesn't produce the warning.

So something is going wrong in the event creation in [MultiBackend.js](https://github.com/LouisBrunner/react-dnd-multi-backend/blob/master/src/lib/MultiBackend.js#L94), but I can't figure what exactly.

**Comment by haf** [comment]
Thank you for investigating. I'll let you know if we find the problem ourselves, in this issue.

**Comment by tom-s** [comment]
Hi i have the same issue and it seems to be more than just a warning as it prevents me from dragging.
It seems to happen quite randomly and I can't figure out how to properly reproduce the issue / fix it. Have you found something @haf ?

**Comment by haf** [comment]
@tom-s You probably have another issue, since it prevents you from dragging. However, it should be said that this repo's dependencies have moved from v2.x to v5.x since this issue was created and I haven't done the upgrade yet; so you're likely experiencing a different code path than I.

**Comment by LouisBrunner** [comment]
@tom-s As @haf said it seems like another issue, could you post a small example (or create another issue) so we can help?

@haf The latest version can technically be used with any version of `react` or `react-dnd` (the requirements on `v5` were an experiment to make the package work with the latest dependencies and should have never been released).

**Comment by haf** [comment]
@LouisBrunner Do you think I can just upgrade without breaking my code then? I've been trying to find docs for the three major versions, but should I take your comment to mean, there's no major API changes to speak of?

**Comment by LouisBrunner** [comment]
@haf The only noticeable change should be your dependencies: `react`, `react-dom`, `react-dnd`, `react-dnd-html5-backend` and `react-dnd-touch-backend` are now `peerDependencies` so you'll need to add them in your project `dependencies` in case you don't have them ([see here](https://github.com/LouisBrunner/dnd-multi-backend/tree/next/packages/react-dnd-multi-backend#migrating-from-312)).

Some changes were made to make the library work with `react-dnd` v5 and `react` v16, but that shouldn't prevent the older versions from working. No user-facing API changed.
